src/xlat-i/xi-Entry.C

```diff
@@ -885,31 +882,20 @@ void Entry::genGroupDefs(XStr& str) {
             << "  _TRACE_BEGIN_EXECUTE_DETAILED(CpvAccess(curPeEvent),ForBocMsg,(" << epIdx()
             << "),CkMyPe(),0,NULL, NULL);\n";
       if (isAppWork()) str << " _TRACE_BEGIN_APPWORK();\n";
-      str << "#if CMK_LBDB_ON\n"
-             "  // if there is a running obj being measured, stop it temporarily\n"
-             "  LDObjHandle objHandle;\n"
-             "  int objstopped = 0;\n"
-             "  LBManager *the_lbmgr = (LBManager *)CkLocalBranch(_lbmgr);\n"
-             "  if (the_lbmgr->RunningObject(&objHandle)) {\n"
-             "    objstopped = 1;\n"
-             "    the_lbmgr->ObjectStop(objHandle);\n"
-             "  }\n"
-             "#endif\n";
       str << "#if CMK_CHARMDEBUG\n"
              "  CpdBeforeEp("
           << epIdx()
           << ", obj, NULL);\n"
              "#endif\n  ";
+      str << "CkCallstackPush((Chare*)obj);\n  ";
```

Comment:
This won't start load timers for the group since its location record will be a `nullptr`, right?

Reply:
Correct. It will, however, suspend timing of the previous object -- resuming it after the call to CkCallstackPop.